src/sentry/conf/server.py

```diff
@@ -1176,7 +1176,9 @@ def create_partitioned_queues(name):
 # when checking REMOTE_ADDR ip addresses
 SENTRY_USE_X_FORWARDED_FOR = True
 
-SENTRY_DEFAULT_INTEGRATIONS = ()
+SENTRY_DEFAULT_INTEGRATIONS = (
+    'sentry.integrations.slack.SlackIntegration',
```

Comment:
is this built in now?

Reply:
@mitsuhiko yeah i was using this as a way to test defaults, but we decided there was no point in maintaining first-party integrations in a different repo so just adding it to the settings (we'll create an entry point for external ones)